.githooks/pre-commit

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+SWIFTLINT=./Pods/SwiftLint/swiftlint
+CONFIG=.swiftlint.yml
+
+if ! command -v "${SWIFTLINT}" &> /dev/null; then
+	echo "${SWIFTLINT} is not installed. Please run 'make install-swiftlint'."
```

Comment:
nit: This error is no longer accurate. You could change it to "Please run 'pod install'."